oeps/oep-0019-bp-developer-documentation.rst

```diff
@@ -302,6 +319,10 @@ Best practices
 Learning rST
 ^^^^^^^^^^^^
 
+* We have a `cheatsheet comparing rST to Markdown`__.
```

Comment:
One more thing: do you think it would be useful to have a back reference from the `Markdown Format` section back to this?  So folks reading about Markdown specifically know about our effort to create this cheatsheet?